template/sample/MauiSample.csproj

```
@@ -69,4 +69,12 @@
         <ProjectReference Include="..\android\NewBinding.Android.Binding\NewBinding.Android.Binding.csproj" />
     </ItemGroup>
 
+	<!-- Reference the Android binding dependencies -->
+	<!-- Uncomment the code block below and update the AndroidLibrary path to point your library .aar -->
+	<!-- <ItemGroup Condition="$(TargetFramework.Contains('android'))">
+        <AndroidLibrary Include="..\android\native\newbinding\build\outputs\deps\{yourNewBindingLibrary.aar}">

```

Made a similar comment on the docs, I think it may be helpful to update the placeholder name here to something like `yourDependency` or `yourDependencyLibrary` for clarity -- dependencies from the library being bound (rather than the library/.aar you are creating with the binding) may need to be added here
 
```suggestion
        <AndroidLibrary Include="..\android\native\newbinding\build\outputs\deps\{yourDependencyLibrary.aar}">

```

---

renaming to yourDependencyLibrary to be consistent with your suggestion for the docs as well!